### PR TITLE
Proposed Modus operandi for RAP Champions

### PR DESCRIPTION
**Pull request by gomezcatalina**
Starting a document to articulate what we do, how we do it, how we implement it, test it and improve it: fostering a culture of progress over perfection

 :basecampy:

### CODE REVIEW

**Review by stoyelq — APPROVED** [review]
Looks great!

**Review by gomezcatalina — COMMENTED** [review]
More progress! Yeah! :sparkles:

File: reports/Modus-operandi-RAP.Rmd [hunk 1]
```diff
@@ -0,0 +1,97 @@
...
+knitr::opts_chunk$set(echo = TRUE)
+# requires R package `rmdformats`
+```
+
+## The Strategic Reproducible Analytical Pipeline (RAP) Initiative
+
+The Strategic Science Planning and Program Integrity (SSPPI) division in DFO Maritimes supports various projects and presents many opportunities to modernize data management and reporting practices. In 2018, requests to the division to identify and summarize the available DFO and non-DFO datasets within the Maritimes Region were becoming increasingly frequent. Born out of frustration and a need for swift and effective approaches, a team of self-proclaimed Strategic Reproducible Analytical Pipeline (RAP) Champions was formed to automate the creation of these Reports. The primary objective of this initiative has been to develop a web-based tool to generate Reproducible Reports to identify and describe DFO and non-DFO datasets within a user-defined area. Specifically, this project address internal requests that support processes that provide frequent and standardized advice, such as CSAS, Marine Spatial Planning, Aquaculture Siting Responses, and Environmental Response, which have an emphasis on Species at Risk.
```
Comment (remi-daigle) [inline reports/Modus-operandi-RAP.Rmd:21]: ``
```suggestion
The Strategic Science Planning and Program Integrity (SSPPI) division in DFO Maritimes supports various projects and presents many opportunities to modernize data management and reporting practices. In 2018, requests to the division to identify and summarize the available DFO and non-DFO datasets within the Maritimes Region were becoming increasingly frequent. Born out of frustration and a need for swift and effective approaches, a team of self-proclaimed Strategic Reproducible Analytical Pipeline (RAP) Champions was formed to automate the creation of these Reports. The primary objective of this initiative has been to develop a web-based tool to generate Reproducible Reports to identify and describe DFO and non-DFO datasets within a user-defined geographic area. Specifically, this project address internal requests that support processes that provide frequent and standardized advice, such as CSAS, Marine Spatial Planning, Aquaculture Siting Responses, and Environmental Response, which have an emphasis on Species at Risk.

```

**Comment by gomezcatalina** [comment]
Thank you for the feedback - let's continue the conversation to make this an evergreen document! :jigsaw:

File: reports/Modus-operandi-RAP.Rmd [hunk 2]
```diff
@@ -0,0 +1,98 @@
...
+## Strategic RAP Champions: who are we?
+
+This section is a modified version of the [RAP champions network in the UK](https://gss.civilservice.gov.uk/about-us/champion-networks/reproducible-analytical-pipeline-rap-champions/).
+
+Strategic RAP champions support the implementation of reproducible analytical pipelines at DFO. This includes:
+
+-   Promoting reproducible analysis (i.e. analysis with a clear audit trail that explains how and why it was carried out),
+-   Actively using reproducible analytical pipelines (i.e. the software methods used to make analysis reproducible),
+-   Sharing knowledge and providing advice and support to members of the analytical community who want to learn about, and implement, reproducible analysis and reproducible analytical pipelines.
+-   Using platforms like GitHub or GItLab to share code so others can learn, use and/or adapt what has been done.
```
Comment (remi-daigle) [inline reports/Modus-operandi-RAP.Rmd:43]: G**i**tLab

**Review by remi-daigle — COMMENTED** [review]
Looks great! I just fixed a couple typos!

**Pull request merged**